homeassistant/components/group/registry.py

```diff
@@ -8,14 +8,60 @@
 from dataclasses import dataclass
 from typing import Protocol
 
-from homeassistant.const import STATE_OFF, STATE_ON
+from homeassistant.components.climate import HVACMode
+from homeassistant.const import (
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_ARMED_CUSTOM_BYPASS,
+    STATE_ALARM_ARMED_HOME,
+    STATE_ALARM_ARMED_NIGHT,
+    STATE_ALARM_ARMED_VACATION,
+    STATE_ALARM_TRIGGERED,
+    STATE_CLOSED,
+    STATE_HOME,
+    STATE_NOT_HOME,
+    STATE_OFF,
+    STATE_ON,
+    STATE_OPEN,
+)
 from homeassistant.core import HomeAssistant, callback
 from homeassistant.helpers.integration_platform import (
     async_process_integration_platforms,
 )
 
 from .const import DOMAIN, REG_KEY
 
+EXCLUDED_DOMAINS = {"air_quality", "sensor", "weather"}
+
+ON_OFF_STATES = {
+    "alarm_control_panel": (
```

Comment:
I think we can use the platform str enum here

Reply:
Used the enum where possible. The `person` and `plant` platform have no enum instance.